components/node/eth/output.go

```diff
@@ -39,20 +39,24 @@ func (o *OutputResponse) ToPublicInput(ChainId *big.Int) (bindings.TypesPublicIn
 	if o.NextBlock == nil {
 		return bindings.TypesPublicInput{}, ErrBlockIsEmpty
 	}
+	var withdrawalsRoot common.Hash
+	if o.NextBlock.WithdrawalsHash != nil {
+		withdrawalsRoot = *o.NextBlock.WithdrawalsHash
+	}
 	txHashes := make([][32]byte, len(o.NextTransactions))
 	for i, tx := range o.NextTransactions {
 		txHashes[i] = tx.Hash()
 	}
 	return bindings.TypesPublicInput{
-		Coinbase:         o.NextBlock.Coinbase,
+		BlockHash:        o.NextBlockRef.Hash,
+		ParentHash:       o.BlockRef.ParentHash,
 		Timestamp:        o.NextBlock.Time,
 		Number:           o.NextBlock.Number.Uint64(),
-		Difficulty:       common.Big0,
 		GasLimit:         new(big.Int).SetUint64(o.NextBlock.GasLimit),
 		BaseFee:          o.NextBlock.BaseFee,
-		ChainId:          ChainId,
```

Comment:
We don't need `ChainId` arg anymore :)

Reply:
Thank you for noticing me :) I removed them!